src/main.rs

```diff
@@ -28,6 +28,8 @@ struct Args {
     trace: Option<PathBuf>,
     #[structopt(long = "--memory_file")]
     memory_file: Option<PathBuf>,
+    #[structopt(long = "--layout", default_value = "plain")]
+    layout: String,
```

Comment:
Here we can check that the layout we receive is one of the 6 defined layouts.

Reply:
I added a validation function to the argument (6990cf9)